src/vision/status_quo/alan_started_trusting_the_rust_compiler.md

```diff
@@ -0,0 +1,104 @@
+# 😱 Status quo stories: Alan started trusting the Rust compiler
```

Comment:
This title made me think this story would be a happy one with Alan reaching a point where he trusts the compiler. In fact, the opposite is true. 

Perhaps another title would be more descriptive: "Multiple Async Runtimes Erode Alan's Trust in the Rust Compiler"

Reply:
I understand what you mean, but I think that your suggestion misses the build up of trust that I personally went through.
What do you think about:
* "Alan started trusting the Rust compiler, but then he encountered async"
or 
* "Alan started trusting the Rust compiler, but then... async"

I think that that captures the build up as well as already showing that this isn't going to be a story with a happy ending

Reply:
I like those

Reply:
I'd run with the second one, myself

Reply:
Remember the story isn't over, this is just where async is now. Alan trusts there will a shiny future where async programming is as gotcha free as it can be.